app/models/region.rb

```diff
@@ -54,6 +54,36 @@ def log_payload
     attrs.symbolize_keys
   end
 
+  def syncable_patients
+    case region_type
```

Comment:
Is this conditional meant to be a temporary fix?

Reply:
Why not apply the block definition to all regions? Is performance a concern?

Reply:
I mentioned this [earlier](https://github.com/simpledotorg/simple-server/pull/1333#discussion_r522784347) and it might have gotten lost in the thread. I think this is the whole purpose of the PR, the rest is just structural changes. 

Block-level sync is not useful if we're just syncing registered patients. We **cannot** launch block-level sync without the neighboring queries.

Reply:
Summary:

* We need this, cannot launch without it
* If we start doing this for all other regions, we shouldn't blindly do it every everything else, the sync boundaries for blocks happen to be quite atypical, this may not be the case for others. We should discuss and then implement. Cordoning this off is a safer bet.
* We should also think twice before doing it for bigger regions because of perf reasons, this will fly for blocks because you're starting very narrow, but it won't work for States, that's too much data.

Reply:
> We cannot launch block-level sync without the neighboring queries.

Agree. To clarify, my comment here was to ask if we can apply the "unioned" definition across the board, but the conditional here makes the change _more_ surgical. It preserves current behavior for facility groups, and only changes the sync definition for blocks. Let's leave as is.